iot-hub/Quickstarts/read-d2c-messages/read_device_to_cloud_messages_async.py

```diff
@@ -0,0 +1,83 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+This sample demonstrates how to use the Microsoft Azure Event Hubs Client for Python async API to 
+read messages sent from a device. Please see the documentation for @azure/event-hubs package
+for more details at https://pypi.org/project/azure-eventhub/
+
+For an example that uses checkpointing, follow up this sample with the sample in the 
+azure-eventhub-checkpointstoreblob package on GitHub at the following link:
+
+https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/samples/receive_events_using_checkpoint_store_async.py
+"""
+
+
+import asyncio
+from azure.eventhub import TransportType
+from azure.eventhub.aio import EventHubConsumerClient
+
+
+# Event Hub-compatible endpoint
+# az iot hub show --query properties.eventHubEndpoints.events.endpoint --name {your IoT Hub name}
+EVENTHUB_COMPATIBLE_ENDPOINT = "{your Event Hubs compatible endpoint}"
+EVENTHUB_COMPATIBLE_ENDPOINT = "iothub-ns-yijun-ioth-3130889-4398262da5.servicebus.windows.net"
```

Comment:
Don't forget to yank your hard-coded constants :)

Reply:
This is embarrassing. I used them to test the script. 
Not saved file before git commit in VS code again.
I regenerated the key on portal as well since it's compromised.
Thanks for pointing this out.

Reply:
Just realized the resource was already deleted on github.

Reply:
I've done the same thing roughly a million times :)